### ISSUE TITLE
search or play puts in two FF's on the end

### ISSUE THREAD

**Issue by ranger786** [julianh2o/RokuAlexaLambdaSkill#21]
Alexa, tell roku to play titanic 
Alexa, tell roku to play king kong on plex

in all the searches it puts the extra "FF's" at the end, how to fix this?

Thanks

**Comment by julianh2o** [comment]
Hmm, you might want to look into line 200 in https://github.com/julianh2o/RokuAlexaLambdaSkill/blob/master/RokuControlServer/server.js

That's where the keypresses for searching and playing are coming from. Is it possible that it's not tabbing over enough when it's trying to go to the results? Has there been an update in how the search layout operates?

Look at that sequence there and see if you can follow along with it. The line that goes `createTypeSequence(text)` is the one that's actually simulating the correct letters. Perhaps the series of "right" pushes below it are doing something weird?

Unfortunately it'll be pretty hard for me to debug without seeing exactly what's happening for you.. I'd tinker with that code (and relaunch the server each time you edit it) and see if you can get something working!

**Comment by cmon69** [comment]
I'm getting that two!

**Comment by cmon69** [comment]
ranger786, did you find a solution? I didn't want to mess with the code but am thinking of trying it after taking out 

rokuAddress+"keypress/right",
keyDelay,
rokuAddress+"keypress/right",
keyDelay,

**Comment by julianh2o** [comment]
Yeah, definitely give that a shot.. though it's also possible that you'll
need to add a duplicate of those lines.

Basically what you're seeing there in the code is the exact key presses
that the program is simulating..

Another option would be to record a quick video of the behavior you're
seeing, so that I can help you adjust the code a bit better.

**Comment by cmon69** [comment]
Here is the video, also I wanted to mention that I was having the same issue using the scripts that I had previously use from https://github.com/chris1642/alexaroku if it helps. https://www.youtube.com/watch?v=gFcHfBSAyvw&feature=youtu.be

**Comment by julianh2o** [comment]
Sorry for the delay, got swept up in something!

I think you should be able to fix this by inserting a new line after line 221 if server.js.

Line 221 reads: `],createTypeSequence(text),[`

If you add a new line after that something like..

     1000,

Or to be extra safe...

    2000,

It may work for you.

Looks like you might want to also increase the delay on line 220 from 500 to something like 1000.

Let me know if that helps!
